main.go

```diff
@@ -152,20 +184,49 @@ func main() {
 	log.Infoln("Starting blackbox_exporter", version.Info())
 	log.Infoln("Build context", version.BuildContext())
 
-	yamlFile, err := ioutil.ReadFile(*configFile)
-	if err != nil {
-		log.Fatalf("Error reading config file: %s", err)
+	if err := sc.reloadConfig(*configFile); err != nil {
```

Comment:
Yeah, it will be loaded at the first time, and then, just fire **ONLY** by signal/POST.